src/NuGet.Core/NuGet.Frameworks/FrameworkConstants.cs

```diff
@@ -188,6 +189,7 @@ public static class CommonFrameworks
 
             // .NET 5.0 and later has NetCoreApp identifier
             public static readonly NuGetFramework Net50 = new NuGetFramework(FrameworkIdentifiers.NetCoreApp, Version5);
+            public static readonly NuGetFramework Net60 = new NuGetFramework(FrameworkIdentifiers.NetCoreApp, Version6);
```

Comment:
Can you add it to the fast parsing switch statement in NuGetFrameworkFactory 

https://github.com/NuGet/NuGet.Client/blob/1a7aa2dd343562168a221b1da959f1704a784e32/src/NuGet.Core/NuGet.Frameworks/NuGetFrameworkFactory.cs#L687